README.md

```diff
@@ -172,3 +182,21 @@ the API documentation in [toxcore/tox.h](toxcore/tox.h) for more information.
 
 - [Another echo bot](https://wiki.tox.chat/developers/client_examples/echo_bot)
 - [minitox](https://github.com/hqwrong/minitox) (A minimal tox client)
+
+## SAST Tools
+
+- [clang-tidy](https://clang.llvm.org/extra/clang-tidy/): A clang-based C++
+  "linter" tool.
+- [Coverity](https://scan.coverity.com/): Find and fix defects in your Java,
+  C/C++, C#, JavaScript, Ruby, or Python open source project for free.
+- [cppcheck](https://cppcheck.sourceforge.io/): A static analysis tool for C/C++
+  code.
+- [cpplint](https://github.com/cpplint/cpplint): Static code checker for C++
```

Comment:
`clang-tidy` is a "linter" tool, but `cpplint` is a checker. You'd think they would be described using the same terminology, whether as linter or checker.

Reply:
Copied from their respective websites, so whatever they claim themselves to be. What do you want it to say instead?

Reply:
I'm pointing out the inconsistency that they are the same type of a tool, yet we use two "synonyms" for it (for the type of a tool they are) in the same list: one is being called "linter" and another "checker", when sticking with just one terminology would have been more consistent. So just make both use either "linter" or "checker".

Eh, maybe I'm just being too pedantic. You already fixed the more egregious things, like missing articles, inconsistent capitalization and one tool having oddly inconsistent description sentence format in comparison to others. This one, while somewhat inconsistent, is arguably fine, so I will approve the PR without it being addressed.